sample/build.sbt

```diff
@@ -0,0 +1,15 @@
+import Defaults._
+import de.johoop.testngplugin.TestNGPlugin
+
+lazy val root = (project in file("."))
+  .settings(
+    name := "Sample",
+    version := "2.5.0-SNAPSHOT",
+    scalaVersion := "2.11.11",
+    crossScalaVersions := Seq("2.10.6", "2.11.11", "2.12.4"),
+    libraryDependencies ++= Seq(
+      "com.linkedin.play-testng-plugin" %% "play-testng-helpers" % "2.5.0"
+    )
+  )
+  .enablePlugins(play.sbt.PlayJava)
+  .enablePlugins(TestNGPlugin)
```

Comment:
Why is this needed?

Reply:
Without this, the TestNG-based tests weren't running in sbt. For Scala 2.11, I had to update from 3.0.2 to 3.1.1 to get it to build.

Note that I ported the whole thing from the old `Build.scala` to `build.sbt`.

Reply:
Just verified: without enabling the TestNGPlugin, sbt doesn't see or execute the TestNG tests.